server/RenderContext.cs

```diff
@@ -466,8 +466,13 @@ public void Render(XGraphics graphics)
                     if (styles.microBorders.visible)
                     {
                         if (styles.fillMicroBorders)
+                        {
                             DrawMicroBorders(BorderLayer.Fill);
+                            DrawRegions(BorderLayer.Fill);
```

Comment:
I currently have it as always border, but _sometimes_ fill - it might be better to be the other way around?